app/app.go

```diff
@@ -25,12 +26,22 @@ var (
 	tpool *pool.Pool
```

Comment:
I can't put this into app because it has dependencies in a bunch of other places. So I say we merge, wait for the App refactor on Chad's end, and then pull the variables into the App type. Either that or we merge this and Chad does it as part of his App refactor PR.